README.md

```diff
@@ -6,79 +6,67 @@ Welcome to the GitHub Gems project! This project hosts a data analytics pipeline
 
-1. [Create a new repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) in your Github account and name it `github-stars-pipeline`.
+### Git Clones
+Git clones are essential metrics that show how many times a repository has been duplicated, indicating its influence and reach within the developer community.
 
-2. Clone this repo.
+### Package Downloads
+Package downloads are significant metrics that reflect the number of times a software package has been downloaded, demonstrating its usage and popularity.
 
-```bash
-git clone https://github.com/edsioufi/github-stars-pipeline.git
-```
+# What is the source of our data?
+For our project we need to have a way to access all open source repositories' data. We have a few options, but we are most interest in two of them, GitHub API and GH Archive. 
 
+Lets discuss the difference between them, the pros and cons.
 
-3. Point your local clone to your own remote (so that you can modify your copy of the repo, not the template). Make sure you repalce `{your_github_username}` with the corresponding value.
+### Advantages of the Github API
+* Most recent fresh data
+* Speed and performance
+* Data accuracy
+* Not a third party 
 
-```bash
-cd github-stars-pipeline
-git remote set-url origin https://github.com/{your_github_username}/github-stars-pipeline.git
-```
+### Disatvantages of the Github API
+* Primary rate limit for requests (5000 requests per hour for authorized users and 15000 requests per hour for enterprise users)
+* Complex load logic
+* Not primarily focused on archived data
 
-4. Push to your new github repo.
+### Advantages of GH Archive
+* Offers a comprehensive, historical log of GitHub events dating back to 2011.
+* Free to use and easily accessible for downloading historical GitHub event data.
+* Provides access to a massive amount of data because it records all public GitHub events.
 
-```bash
-git push origin master
-```
+### Disatvantages of GH Archive
+* Third party source
+* Can have slower speed compared to Github API
+* No real time data
 
-### Set up your python environment and DuckDB
+GH Archive is an invaluable resource for historical data analysis and large-scale trend observations on GitHub projects. However, its use involves significant data handling, processing demands, and potential costs, making it less suitable for real-time applications, small-scale needs, or users without substantial data processing capabilities. For tasks requiring real-time data or targeted queries, the GitHub API may be a more appropriate choice.
 
-1. Create a python virtual environment for your repo:
+In our case, it would be the best to work with GH Arhcive, simply because there are no request limitations. 
 
-```bash
-python -m venv venv
-source venv/bin/activate
-```
+### What format is our data delivered in?
```

Comment:
Here let's mention instead the deliverable data format, which will be data tables that can be queried using SQL.